packages-exp/auth-exp/src/core/persistence/in_memory.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Persistence, PersistenceType, PersistenceValue } from '../persistence';
+
+class InMemoryPersistence implements Persistence {
+  type: PersistenceType = PersistenceType.NONE;
+  storage: {
+    [key: string]: PersistenceValue;
+  } = {};
+
+  async isAvailable(): Promise<boolean> {
+    return true;
+  }
+
+  async set(key: string, value: PersistenceValue): Promise<void> {
+    this.storage[key] = value;
+  }
+
+  async get<T extends PersistenceValue>(key: string): Promise<T | null> {
```

Comment:
do you have a version of this one with instantiator?

Reply:
See comment above.